### PR TITLE
Task/cb cyg mysql

### PR DESCRIPTION
**Pull request by andreattid**

### CODE REVIEW

File: tests/common/test_utils.py [hunk 1]
```diff
@@ -42,7 +43,7 @@ def initialize_log(log_level='INFO'):
     # eval(__logger__+".log_level"+"("+"#>> Test_utils: [LOG] Initialized"+")", log_level)
 
 
-@staticmethod
+#@staticmethod
```
Comment (XavierVal) [inline tests/common/test_utils.py]: why do you comment the @staticmethod??

File: tests/properties.json [hunk 2]
```diff
@@ -52,6 +52,23 @@
...
     },
+    "MYSQL": {
+      "cygnus_path": "/notify",
+      "cygnus_port": "5050",
+      "cygnus_protocol": "http",
+      "cygnus_version_port": "8081",
+      "cygnus_version_path": "/version",
+      "cygnussync": "column",
+      "defaultservice": "test",
+      "heartbeat": "",
```
Comment (XavierVal) [inline tests/properties.json]: we should reduce the amount of params that are currently are deprecated

Comment (andreattid) [inline tests/properties.json]: Fixed in 08228f1

File: tests/common/test_utils.py (hunk 1 above)
Comment (andreattid) [inline tests/common/test_utils.py]: NTC, static method is not callable from outer functions and it has to be commented in these cases

Comment (dmoranj) [inline tests/common/test_utils.py]: Why don't you remove it then?

File: tests/steps/cb_steps.py [hunk 3]
```diff
@@ -0,0 +1,174 @@
...
+        duration: in seconds
+        notify_type: ONCHANGE | ONTIME
+        notif_cond_values: list of attributes separated by semi-colon (;) or interval e.g. PT1S
+        throttling: time limit for the notification
+        """
+
+        assert_in(context.table[0]['notify_type'], ["ONTIME", "ONCHANGE"], "[ERROR] Invalid subscription type")
+
+        # Retrieve the service and subservice real value and give to the CB object
+        #TestUtils.set_service_and_subservice(context, context.table[0]["service"], context.table[0]["subservice"])
```
Comment (dmoranj) [inline tests/steps/cb_steps.py]: Why this line is commented?

File: tests/steps/cb_steps.py [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+
+    __logger__.info('Added entity to the list of entities to be removed')
+
+    # Launch the requests to CB
+    resp = context.o['CB'].standard_entity_creation(payload)
+    eq_(200, resp.status_code)
+    cb_content_json = json.loads(resp.content)
+    __logger__.info(cb_content_json)
+    #TestUtils.
+    remember(context, "cb_response", resp)
```
Comment (dmoranj) [inline tests/steps/cb_steps.py]: Add the final line before German tell you to do it :D

File: tests/steps/cb_steps.py (hunk 3 above)
Comment (XavierVal) [inline tests/steps/cb_steps.py]: Unknown reason, removed.
Fixed in 9fe2660

File: tests/steps/cb_steps.py (hunk 4 above)
Comment (XavierVal) [inline tests/steps/cb_steps.py]: FIxed in 9fe2660

File: tests/common/test_utils.py (hunk 1 above)
Comment (XavierVal) [inline tests/common/test_utils.py]: Removed all the commented tags
Fixed in af29b1e

**Pull request merged**